README.md

```diff
@@ -2,16 +2,16 @@
  The GitHub API is documented here: https://docs.github.com/en/rest
 
  We will be connecting to this API to create a version of 'github'.
- You should clone this repo and in terminal type sbt run, in your browser on localhost:9000 you should see a page with the title "Welcome to your version of GitHub".
+ //You should clone this repo and in terminal type sbt run, in your browser on localhost:9000 you should see a page with the title "Welcome to your version of GitHub".
  
-  For package structure use standard: view (html templates), controller (API endpoint entry), service (Orchestration, should be light weight seeing as    we're only calling one endpoint), connector (To speak to api.github), model (Domain model data representation).
+  //For package structure use standard: view (html templates), controller (API endpoint entry), service (Orchestration, should be light weight seeing as    we're only calling one endpoint), connector (To speak to api.github), model (Domain model data representation).
```

Comment:
Could we maybe rephrase and ask them to create the packages views, controllers, ... and then explain what they mean after in bullet points? And maybe rephrase "(To speak to api.github)" to send requests to external services or something?